k9mail/src/main/java/com/fsck/k9/ui/settings/general/GeneralSettingsFragment.kt

```diff
@@ -112,7 +112,7 @@ class GeneralSettingsFragment : PreferenceFragmentCompat() {
         initializeExtraKeysManagement()
         initializeGlobalpEpKeyReset()
         initializeAfterMessageDeleteBehavior()
-        initializeGlobalPlanckSync()
+        //initializeGlobalPlanckSync()
```

Comment:
as for the xml code it might have some sense to comment, but for the Kotlin/Java let's make a code based on the configuration or even a simple boolean flag rather than simply commenting the code which not making git meaningless but making codebase less readable and clean.

Reply:
ok, I enable it only on endUser variant.